source/guides/guides/cross-browser.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Cross-Browser
+---
+
+Cypress has the capability to run tests across multiple browsers. Currently, Cypress supports Firefox and Chrome-family browsers (including Electron).
+
+{% note warning 'Web Security' %}
+Tests that require {% url "`chromeWebSecurity` configuration option to be disabled" web-security#Disabling-Web-Security %} will not run in non-Chrome family browsers.
+{% endnote %}
+
+
+Excluding Electron, the desired browsers for testing need to be installed on your local system or CI environment to be detected and utilized by Cypress. A full list of detected browsers will be displayed within the browser selection menu of the Test Runner.
+
+{% imgTag /img/guides/cross-browser/cypress-browser-selector.png "Cypress Test Runner with Firefox selected as the browser" "no-border" %}
+
+The desired browser can also specified via the {% url `--browser` command-line#Options %} flag when using {% url `open` command-line#cypress-open %} or {% url `run` command-line#cypress-run %} commands to launch Cypress. For example, to launch Cypress with Firefox as the default selection:
+
+```shell
+cypress open --browser firefox
+```
+
+To make launching of Cypress with a specific browser even more convenient, npm scripts can be used as a shortcut:
+
+```json
+"scripts": {
+  "cy:open:chrome": "cypress open --browser chrome",
+  "cy:open:firefox": "cypress open --browser firefox",
+  "cy:run:chrome": "cypress run --browser chrome",
+  "cy:run:firefox": "cypress run --browser firefox"
+}
+```
+
+# Continuous Integration Strategies
+
+When incorporating testing of multiple browsers within your QA process, you must implement a CI strategy that provides an optimal level of confidence while taking into consideration test duration and infrastructure costs. The optimal strategy will vary by the type and needs of a particular of project, and in this guide we present several strategies to consider when crafting the right strategy for your project.
+
+CI strategies will be demonstrated with the [Circle CI Cypress Orb](https://circleci.com/orbs/registry/orb/cypress-io/cypress) for its concise and readable configuration, but the same concepts apply for most modern CI providers.
+
+{% note info 'Docker Images for Testing' %}
+The CI configuration examples within this guide utilize [Cypress maintained Docker images](https://github.com/cypress-io/cypress-docker-images/tree/master/browsers) to provision testing environments with desired versions of Node, Chrome, and Firefox.
+{% endnote %}
+
+## Periodic Basis
```

Comment:
I don't particularly like the titles of each section, it didn't seem obvious to me if I was searching here what each section actual meant to me. 

My suggestion is to move the 'ignore' section below the 'some tests' section (not sure why they're split up since the seem to relevant) and to rename the titles. 

Also I was confused like, what is the different between 'Subset of tests' and 'Running specific tests by Browser'?? until I read the section itself

**Before**

<img width="238" alt="Screen Shot 2019-12-12 at 4 39 03 PM" src="https://user-images.githubusercontent.com/1271364/70703189-f327d900-1cfd-11ea-88dc-e38b3af88835.png">


**After**

<img width="224" alt="Screen Shot 2019-12-12 at 4 36 55 PM" src="https://user-images.githubusercontent.com/1271364/70703103-c83d8500-1cfd-11ea-9c86-00b17dfe1a33.png">

```suggestion
## Only run periodically
```

Reply:
To further clarify "Subset of Tests" should be "Subset of Test Files." These two sections are very different since one is talking about spec files and the other is talking about individual tests.

Reply:
hmmm, yah, the whole specs versus tests thing, yah, maybe just say 'Only run some files', maybe even avoid the word test.